tools/init-tfvars.sh

```diff
@@ -78,35 +88,51 @@ if test $GOOGLE_PROVIDER_COUNT -ne 0; then
-    printf "gcp_project_id=\"${GCP_PROJECT_ID}\"\n\n" >> $TFVARS_FILE
-    printf "\tgcp_project_id=${BLUE}\"${GCP_PROJECT_ID}\"${NC}\n"
 
-    printf "# GCP project in which OAuth clients for Google Workspace connectors will be provisioned\n" >> $TFVARS_FILE
-    printf "#  - if you're not connecting to Google Workspace data sources, you can omit this value\n" >> $TFVARS_FILE
-    printf "google_workspace_gcp_project_id=\"${GCP_PROJECT_ID}\"\n\n" >> $TFVARS_FILE
-    printf "\tgoogle_workspace_gcp_project_id=${BLUE}\"${GCP_PROJECT_ID}\"${NC}\n"
+    [[ -f variables.tf ]] && grep -q '^variable "gcp_project_id"' variables.tf
+    if [[ $? -eq 0 ]]; then
+      printf "gcp_project_id=\"${GCP_PROJECT_ID}\"\n\n" >> $TFVARS_FILE
+      printf "\tgcp_project_id=${BLUE}\"${GCP_PROJECT_ID}\"${NC}\n"
+    fi
 
     # tenant SA emails
-    printf "# GCP service account emails in the list below will be allowed to invoke your proxy instances\n" >> $TFVARS_FILE
-    printf "#  - if you're deploying to AWS rather than GCP, you can omit this value\n" >> $TFVARS_FILE
-    printf "#  - for initial testing/deployment, it can be empty list; it needs to be filled only once you're ready to authorize Worklytics to access your data\n" >> $TFVARS_FILE
-    printf "worklytics_sa_emails=[\n" >> $TFVARS_FILE
-    printf "  # put 'Service Account Email' value here, which you can obtain from Worklytics ( https://intl.worklytics.co/analytics/integrations/configuration )\n" >> $TFVARS_FILE
-    printf "]\n\n" >> $TFVARS_FILE
-
-    # example user for Google Workspace
-    printf "# Google Workspace example user \n" >> $TFVARS_FILE
-    printf "#  - this is used to aid testing of Google Workspace connectors against a real account (eg, your own); if you're not using those, it can be omitted\n" >> $TFVARS_FILE
-    GOOGLE_WORKSPACE_EXAMPLE_USER=$(gcloud config get account)
-    printf "google_workspace_example_user=\"${GOOGLE_WORKSPACE_EXAMPLE_USER}\"\n\n" >> $TFVARS_FILE
-    printf "\tgoogle_workspace_example_user=${BLUE}\"${GOOGLE_WORKSPACE_EXAMPLE_USER}\"${NC}\n"
-
-        # example admin for Google Workspace
-    printf "# Google Workspace example admin \n" >> $TFVARS_FILE
-    printf "#  - this is used to aid testing of Google Workspace connectors against a real account, in cases where an admin is explicitly required\n" >> $TFVARS_FILE
-    GOOGLE_WORKSPACE_EXAMPLE_USER=$(gcloud config get account)
-    printf "google_workspace_example_admin=\"${GOOGLE_WORKSPACE_EXAMPLE_USER}\"\n\n" >> $TFVARS_FILE
-    printf "\tgoogle_workspace_example_admin=${BLUE}\"${GOOGLE_WORKSPACE_EXAMPLE_USER}\"${NC}\n"
+    [[ -f variables.tf ]] && grep -q '^variable "worklytics_sa_emails"' variables.tf
+    if [[ $? -eq 0 ]]; then
+      printf "# GCP service account emails in the list below will be allowed to invoke your proxy instances\n" >> $TFVARS_FILE
+      printf "#  - if you're deploying to AWS rather than GCP, you can omit this value\n" >> $TFVARS_FILE
```

Comment:
skip if HOST_PLATFORM === "aws" ? or remove and do the whole block if host === "gcp"
```suggestion
      printf "#  - if you're deploying to AWS rather than GCP, you can omit this value\n" >> $TFVARS_FILE
```